src/Initializer/CellLocalMatrices.cpp

```diff
@@ -537,26 +539,48 @@ void seissol::initializers::initializeDynamicRuptureMatrices( MeshReader const&
           break;
         }
       }
+      /// Traction matrices for "average" traction
```

Comment:
Why "average"? The quotes are a bit scary

Reply:
I don't know. That's Carsten's comment. They are average in the sense the matrix works for all spatial points, as we do not have subcell material resolution?